src/get-started/install/_ios-setup.md

```diff
@@ -86,7 +86,35 @@ and an Apple Developer account. If your app is using Flutter plugins,
 you will also need the third-party CocoaPods dependency manager.
 
 <ol markdown="1">
+<li markdown="1">
+<a name="connect"></a>
+To set up physical device deployment in Xcode, 
+connect your device to the USB port on your 
+computer. 
+</li>
+<li markdown="1">
+<a name="wireless"></a>
+[Optional] To leverage wireless debugging, ensure that 
+your device is on the same network as your computer 
+and that the device has a set passcode.  
+
+While the device is attached, open **Xcode > Window > Devices and Simulators**. 
+Select your phone, and check **Connect via Network.**
 
+<img src="/assets/images/docs/setup/connect-wireless.png" alt="Check the box that says 'connect via network' from the devices and simulators page">
```

Comment:
Can we link to https://help.apple.com/xcode/mac/9.0/index.html?localePath=en.lproj#/devbc48d1bad instead of maintaining our own screenshot we need to update when this UI changes?

Reply:
Sure